entrypoint.sh

```diff
@@ -115,7 +115,16 @@ if [ "$1" = "kubelet" ]; then
         if [ "${RKE_KUBELET_CRIDOCKERD_DUALSTACK}" == "true" ]; then
           EXTRA_FLAGS="--ipv6-dual-stack"
         fi
-        /opt/rke-tools/bin/cri-dockerd --network-plugin="cni" --cni-conf-dir="/etc/cni/net.d" --cni-bin-dir="/opt/cni/bin" ${RKE_KUBELET_PAUSEIMAGE} --container-runtime-endpoint=$CONTAINER_RUNTIME_ENDPOINT ${EXTRA_FLAGS} &
+        STREAM_SERVER_ADDRESS=$(echo "$@" | grep -Eo "\-\-stream-server-address+.*" | awk '{print $1}' | cut -d "=" -f2)
```

Comment:
Is there a PR in RKE that sets `--stream-server-address` from cluster.yml?

Reply:
actually i was working on that and then realised that these options can not be sent as command Args and has to be passed through environment since all the cli options are passed to kubelet command which causes this to fail. 
I have updated it here and also created a rke [PR](https://github.com/rancher/rke/pull/3658) accordingly.

Reply:
ya if you add it in commandArgs, it'd need to be filtered out before starting kubelet. thanks for fixing it in rke!